z2/docs/deposit.md

```diff
@@ -22,21 +23,26 @@ Usage: z2 deposit --chain <CHAIN_NAME> --public-key <BLS_PUBLIC_KEY> --peer-id <
 * `--private-key <PRIVATE_KEY_OF_WALLET_WITH_MIN_10_MIL_FUND>`: The private key of the wallet that has a minimum stake amount of 10 million.
 * `--amount <AMOUNT_IN_MILLION_ZIL>`: The amount in ZIL to deposit. The valid range is from 10 million to 255 million ZIL, allowing a deposit of up to 255 million ZIL.
 * `--reward-address <REWARD_ADDRESS>`: Specifies the address to receive rewards. You can generate a new wallet address to receive the rewards.
-* `--pop-signature <BLS_POP_SIGNATURE>`: The BLS proof-of-possession signature of the validator node.
+* `--signing-address <SIGNING_ADDRESS>`: Specifies the address which signs cross-chain events.
+* `--deposit-auth-signature <DEPOSIT_AUTH_SIGNATURE>`: The BLS proof-of-possession signature of the validator node.
 
 **Note**: The `--private-key` parameter should be the private key of a wallet that has secured a minimum stake amount of 10 million ZILs.
 
 ### Generating Required Values
-To generate the `public-key`, `pop-signature` and `peer-id`, use the following command inside the zq2 folder. Please pass `PRIVATE_KEY_OF_VALIDATOR` to the command input.
+To generate the `public-key`, `deposit-auth-signature` and `peer-id`, use the following command inside the zq2 folder. Please pass `PRIVATE_KEY_OF_VALIDATOR` and `CHAIN_ID` to the command input. 
+
+By default this tool signs over the address derived from the given secret key to generate the `deposit-auth-signature`, you may want to override this for example if deploying via a Delegation contract. To override pass in `"address": "<CONTROL_ADDRESS>"`.
```

Comment:
to have the input parameter clearer to an user, can we call it `control_address`?

@DrZoltanFazekas @JamesHinshelwood

Reply:
Makes sense to make it more specific than `address` especially because there are two other addresses, the reward address and the signing address and people could get easily confused